x-pack/legacy/plugins/maps/public/layers/styles/vector/vector_style.js

```diff
@@ -543,6 +579,15 @@ export class VectorStyle extends AbstractStyle {
     this._iconSizeStyleProperty.syncCircleRadiusWithMb(pointLayerId, mbMap);
   }
 
+  setMBPropertiesForLabelText({ alpha, mbMap, textLayerId }) {
+    mbMap.setLayoutProperty(textLayerId, 'visibility', 'visible');
```

Comment:
Do we need this line? It's overridden later on anyway. 

If so, I'd add a comment as to why this is required.